.github/actions/install-chrome/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Install Chrome browser and driver for Testing
+description: Download and install the compatible Chrome and Chromedriver
+
+inputs:
+  version:
+    description: The version of Chrome and Chromedriver to install.
+    required: false
+    default: 134.0.6998.165 # Ensures compatibility with the testsuite
+
+runs:
+  using: composite
+  steps:
+
+    # Cannot use Chrome for Testing. Acts weirdly and would probably require some changes in the testsuite.
```

Comment:
Just out of curiosity, is it still the case that Chrome for Testing doesn't work with the testsuite on `main`? I remember making this change for 24 but IIRC, it was not the case for `main`?